bayesian/api/api_v2.py

```diff
@@ -60,6 +61,44 @@
 metrics = GunicornPrometheusMetrics(api_v2, group_by="endpoint", defaults_prefix=NO_PREFIX)
 
 
+@api_v2.route('/component-vulnerability-analysis/', methods=['POST'])
+@api_v2.route('/component-vulnerability-analysis', methods=['POST'])
+@login_required
+def component_vulnerability_analysis_post():
+    """Handle the POST REST API call.
+
+    Component Analyses Batch is 3 Step Process:
+    1. Gather and clean Request.
+    2. Query GraphDB.
+    3. Build Stack Recommendation
+    """
+    input_json: Dict = request.get_json()
+    ecosystem: str = input_json.get('ecosystem')
+
+    try:
+        # Step1: Gather and clean Request
+        packages_list, normalised_input_pkgs = ca_validate_input(input_json, ecosystem)
+        # Step2: Get aggregated CA data from Query GraphDB,
+        graph_response = get_batch_ca_vulnerability_data(ecosystem, packages_list)
+        # Step3: Build Unknown packages and Generates Stack Recommendation.
+        stack_recommendation, unknown_pkgs = get_known_unknown_pkgs(
```

Comment:
why do we need unknown package here? i dont think we need to support unknown flow for clair and we dont have SA at all

Reply:
unknown package flow fucntion is being resused here to just build the stack.I am not trigerring ingestion for unknown flow.

Reply:
@yzainee Lets keep it. @mathur07 can modify it accordinly to build stack like the spec.

Reply:
as long as the final version doesnt have any ref to unknown pkg or flow im ok